deploy-theme.sh

```
@@ -0,0 +1,294 @@
+    done
+    exit 1
+else
+    success "Deployment verification passed"
+fi
+
+# Step 7: Clean up old backups (keep last 5)
+log "Step 7: Cleaning up old backups"
+
+if [ -d "$BACKUP_DIR" ]; then
+    # Keep only the 5 most recent backups
+    backup_count=$(ls -1 "$BACKUP_DIR" | grep "$THEME_NAME-backup-" | wc -l)
```

Shellcheck (0.10.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)